app/controllers/application_controller.rb

```diff
@@ -122,8 +122,8 @@ def raise_event(event, payload)
   end
 
   def params_for_create
-    required = api_doc_definition.required_attributes
-    body_params.permit(*api_doc_definition.all_attributes).tap { |i| i.require(required) if required }
+    # We already validate this with OpenAPI validator, that validates every request, so we shouldn't do it again here.
+    body_params.permit!
```

Comment:
@bdunne I am disabling the validation here, since we validate a superset of this for each request, using the openapi validator. (which also validates nested objects, like the `{:extra => {:azure => {:tenant_id => "xy"}}}' that I am adding here)

Reply:
Do we have a test that raises an error when a required field is missing?

Reply:
Hm, not in here. We don't have any required fields in the spec. (I test it in other repo though, so we know it works)

I was trying to add the required fields. But that won't work since we e.g. use the same model in patch query. So first we will need to do a schema refactoring, where post/put/patch will have different readOnly and required definitions,